changelog.json

```diff
@@ -1,6 +1,18 @@
 {
   "repository": "newrelic/node-newrelic",
   "entries": [
+    {
+      "version": "10.3.2",
+      "changes": {
+        "security": [
+          "bump fast-xml-parser and @aws-sdk/client-lambda"
```

Comment:
the security fix was the security agent bumping semver, correct?

Reply:
Should the security agent mention on the new line 8 include a mention of semver explicitly?

Reply:
I don't know. This is a confusing one esp since the client-lambda was only updated in our repo, but due to semver ranges it'd be fine. so really that one doesn't need to be included.

Reply:
also looking at the security agent PR they did more than bump semver https://github.com/newrelic/csec-node-agent/pull/59